docs/userguide/aggregated-api-endpoint.md

```diff
@@ -80,7 +80,7 @@ Or append `/apis/cluster.karmada.io/v1alpha1/clusters/{clustername}/proxy ` to t
 kubectl --kubeconfig karmada-apiserver.config get node
 ```
 
-> Note: For a member cluster that joins karmada in pull mode, we can [deploy apiserver-network-proxy (ANP)](../working-with-anp.md) to access it.
+> Note: For a member cluster that joins karmada in pull mode, if it uses a one-way network, we can [deploy apiserver-network-proxy (ANP)](../working-with-anp.md) to access it.
```

Comment:
`one-way network` means that only member clusters can access the karmada control-plane, am I right.

Is this word well known? Do we need to explain this?

Reply:
> one-way network means that only member clusters can access the karmada control-plane, am I right?

Yes.